config/locales/simple_form.zh-CN.yml

```diff
@@ -8,7 +8,7 @@ zh-CN:
         display_name: 还能输入 <span class="name-counter">%{count}</span> 个字符
         header: 文件大小限制 2MB，只支持 PNG、GIF 或 JPG 格式。图片分辨率将会压缩至 700×335px
         locked: 你需要手动审核所有关注请求
-        note: 还能输入 <span class="name-counter">%{count}</span> 个字符
+        note: 还能输入 <span class="note-counter">%{count}</span> 个字符
```

Comment:
how did we get this wrong?

Reply:
My fault lol - copying & pasting and I didn't notice this until I deploy this to my own instance 🙇